charts/kubearchive/templates/api_server/api_server.yaml

```
@@ -47,7 +47,7 @@ spec:
             - name: POSTGRES_URL
               value: {{ tpl .Values.database.url . }}
             - name: POSTGRES_PORT
-              value: "{{ .Values.database.service.port }}"
+              value: "{{ required "A postgres port must be specified" .Values.database.service.port }}"
```

Since you capitalize "API Server", you should capitalize "Postgres" as well:

```suggestion
              value: "{{ required "A Postgres port must be specified" .Values.database.service.port }}"
```

---

Fix applied